lib/global.g

```diff
@@ -72,6 +72,16 @@ IS_READ_ONLY_GLOBAL := IsReadOnlyGVar;
 
 #############################################################################
 ##
+#F  IS_CONSTANT_GLOBAL ( <name> ) determine if a global variable is constant
+##
+##  IS_CONSTANT_GLOBAL ( <name> ) returns true if the global variable
+##  named by the string <name> is constant and false otherwise (the default)
+##
+
+IS_CONSTANT_GLOBAL := IsConstantGVar;
```

Comment:
Meta-note: It's really crazy that we define kernel function using CamelCase, then define alternate names in SNAKE_CASE, and finally high level names again in CamelCase. (Of course this PR is just imitating what the surrounding code does, so the PR is fine; I just think we may want to resolve this in another PR in the future)